pkg/server/commands/batch_check_command.go

```diff
@@ -40,6 +41,7 @@ type BatchCheckOutcome struct {
 
 type BatchCheckMetadata struct {
 	DatastoreQueryCount uint32
+	DuplicateCheckCount int
```

Comment:
i'm curious what is the goal behind keeping tabs on this number? what are we going to do with this information?